tests/dummy/app/templates/demo/carousel.hbs

```diff
@@ -1,38 +1,41 @@
 <h2>Carousel playground</h2>
 
-<p>You can change some parameters at runtime.</p>
+{{#bs-form model=this as |form|}}
+  <div class="row my-4">
```

Comment:
Seems you are using BS4 markup here for the demo page. But this will still be using BS3 for now!